include/dali/core/bitmask.h

```diff
@@ -48,7 +48,7 @@ class bitmask {
     assert(start >= 0);
     if (start >= size_)
       return size_;
-    auto index = word_idx(start);
+    ptrdiff_t index = word_idx(start);
```

Comment:
Why not `auto` after all?

Reply:
A matter of personal preference, mostly.